app/worker.py

```diff
@@ -21,15 +21,15 @@ def handler(event, context):
             raise Exception('Repository {}/{} does not exist'.format(config.GITHUB_ORG,
                                                                      config.GITHUB_REPOSITORY))
 
-        status_file = repository.file_contents(config.STATUS_FILE, ref='gh-pages')
+        status_file = repository.file_contents(config.STATUS_FILE, ref=config.BRANCH)
```

Comment:
Yes we're going to default to master to keep pages and worker code in one branch. See also https://github.com/blog/2228-simpler-github-pages-publishing